packages/p2p-v2/package.json

```diff
@@ -13,17 +13,18 @@
     },
     "dependencies": {
         "@deriv/api": "^1.0.0",
+        "@deriv/integration": "^1.0.0",
         "@deriv/react-joyride": "^2.6.2",
         "@deriv/utils": "^1.0.0",
-        "@deriv/integration": "^1.0.0",
+        "@sendbird/chat": "^4.9.7",
         "classnames": "^2.2.6",
+        "clsx": "^2.0.0",
```

Comment:
Added `clsx`

Reply:
do we need both clsx and classnames or we can remove classnames 👀

Reply:
yikes forgot to remove classnames